modules/tensor_mechanics/test/tests/substepping/power_law_creep.i

```diff
@@ -0,0 +1,131 @@
+
+[GlobalParams]
+  displacements = 'disp_x disp_y disp_z'
+[]
+
+[Mesh]
+  [gen]
+    type = GeneratedMeshGenerator
+    dim = 3
+    nx = 2
+    ny = 2
+    nz = 2
+  [../]
+  [rename_bnd]
+    type = RenameBoundaryGenerator
+    input = gen
+    old_boundary_name = 'back bottom front left right top'
+    new_boundary_name = 'bottom left top back front right'
```

Comment:
This is actually because I feel the naming convention of the generated block is confusing to me.. I am trying to use the following convention:  
    -x : back, +x: front, -y: left, +y: right, -z: bottom, +z: top
I will add a comment there for clarification.

Reply:
agree with Topher, this will make your input file inconsistent with the rest of MOOSE, which I think is even more confusing.